src/movies.js

```diff
@@ -1,15 +1,2102 @@
+    year: 1995,
+    director: "Mathieu Kassovitz",
+    duration: "1h 38min",
+    genre: ["Crime", "Drama"],
+    rate: 8.1
+  },
+  {
+    title: "8½",
+    year: 1963,
+    director: "Federico Fellini",
+    duration: "2h 18min",
+    genre: ["Drama"],
+    rate: 8.1
+  },
+  {
+    title: "Jaws",
+    year: 1975,
+    director: "Steven Spielberg",
+    duration: "2h 4min",
+    genre: ["Adventure", "Drama", "Thriller"],
+    rate: 8
+  },
+  {
+    title: "Twelve Monkeys",
+    year: 1995,
+    director: "Terry Gilliam",
+    duration: "2h 9min",
+    genre: ["Mystery", "Sci-Fi", "Thriller"],
+    rate: 8
+  },
+  {
+    title: "The Best Years of Our Lives",
+    year: 1946,
+    director: "William Wyler",
+    duration: "2h 50min",
+    genre: ["Drama", "Romance", "War"],
+    rate: 8.1
+  },
+  {
+    title: "Mou gaan dou",
+    year: 2002,
+    director: "Wai-Keung Lau",
+    duration: "1h 41min",
+    genre: ["Crime", "Drama", "Mystery", "Thriller"],
+    rate: 8.1
+  },
+  {
+    title: "Paris, Texas",
+    year: 1984,
+    director: "Wim Wenders",
+    duration: "2h 25min",
+    genre: ["Drama"],
+    rate: 8.1
+  },
+  {
+    title: "The Help",
+    year: 2011,
+    director: "Tate Taylor",
+    duration: "2h 26min",
+    genre: ["Drama"],
+    rate: 8.1
+  },
+  {
+    title: "Faa yeung nin wa",
+    year: 2000,
+    director: "Kar-Wai Wong",
+    duration: "1h 38min",
+    genre: ["Drama", "Romance"],
+    rate: 8.1
+  },
+  {
+    title: "Sholay",
+    year: 1975,
+    director: "Ramesh Sippy",
+    duration: "3h 18min",
+    genre: ["Action", "Adventure", "Comedy", "Drama", "Musical", "Thriller"],
+    rate: 8.2
+  },
+  {
+    title: "Beauty and the Beast",
+    year: 1991,
+    director: "Gary Trousdale",
+    duration: "1h 24min",
+    genre: ["Animation", "Family", "Fantasy", "Musical", "Romance"],
+    rate: 8
+  },
+  {
+    title: "La battaglia di Algeri",
+    year: 1966,
+    director: "Gillo Pontecorvo",
+    duration: "2h 1min",
+    genre: ["Drama", "War"],
+    rate: 8.1
+  },
+  {
+    title: "Ah-ga-ssi",
+    year: 2016,
+    director: "Chan-wook Park",
+    duration: "2h 24min",
+    genre: ["Crime", "Drama", "Mystery", "Romance", "Thriller"],
+    rate: 8.1
+  },
+  {
+    title: "Pirates of the Caribbean: The Curse of the Black Pearl",
+    year: 2003,
+    director: "Gore Verbinski",
+    duration: "2h 23min",
+    genre: ["Action", "Adventure", "Fantasy"],
+    rate: 8
+  },
+  {
+    title: "PK",
+    year: 2014,
+    director: "Rajkumar Hirani",
+    duration: "2h 33min",
+    genre: ["Comedy", "Drama", "Fantasy", "Sci-Fi"],
+    rate: 8.2
+  },
+  {
+    title: "Dog Day Afternoon",
+    year: 1975,
+    director: "Sidney Lumet",
+    duration: "2h 5min",
+    genre: ["Biography", "Crime", "Drama", "Thriller"],
+    rate: 8
+  },
+  {
+    title: "Dead Poets Society",
+    year: 1989,
+    director: "Peter Weir",
+    duration: "2h 8min",
+    genre: ["Comedy", "Drama"],
+    rate: 8
+  }
+];
 
```

Comment:
There was no need to add the data to this file. Jasmine checks it by passing it through the function argument.